### PR TITLE
language-python-update

### PR DESCRIPTION
**Pull request by thomasr8**
# Description
Replaced legacy image reference to next-gen image

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/language-python.md [hunk 1]
```diff
@@ -110,14 +110,14 @@ jobs:
...
-      - image: circleci/python:3.6.4 # every job must define an image for the docker executor and subsequent jobs may define a different image.
+      - image: cimg/python:3.10.1 # every job must define an image for the docker executor and subsequent jobs may define a different image.
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
         environment:
           PIPENV_VENV_IN_PROJECT: true
           DATABASE_URL: postgresql://root@localhost/circle_test?sslmode=disable
-      - image: circleci/postgres:9.6.2 # an example of how to specify a service container
+      - image: cimg/postgres:14.0.0 # an example of how to specify a service container
```
Comment (felicianotech) [inline jekyll/_cci2/language-python.md:120]: `14.0.0` doesn't exist. `14.0` does.

File: jekyll/_cci2/language-python.md [hunk 2]
```diff
@@ -263,15 +263,15 @@ jobs: # A basic unit of work in a run
...
+      - image: cimg/python:3.10.1
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
         environment: # environment variables for primary container
           PIPENV_VENV_IN_PROJECT: true
           DATABASE_URL: postgresql://root@localhost/circle_test?sslmode=disable
       # CircleCI PostgreSQL images available at: https://hub.docker.com/r/circleci/postgres/
-      - image: circleci/postgres:9.6.2
+      - image: cimg/postgres:14.0.0
```
Comment (felicianotech) [inline jekyll/_cci2/language-python.md:274]: `14.0` not `14.0.0`

**Review by felicianotech — CHANGES_REQUESTED** [review]

File: jekyll/_cci2/language-python.md (hunk 1 above)
Comment (thomasr8) [reply jekyll/_cci2/language-python.md]: Fixed

File: jekyll/_cci2/language-python.md (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/language-python.md]: Fixed

**Review by felicianotech — APPROVED** [review]

**Pull request merged**